test/selenium/util/Driver.scala

```diff
@@ -31,8 +33,17 @@ object Driver {
   }
 
   def reset(): Unit = {
+    driver.get(Config.paypalSandbox)
+    driver.manage.deleteAllCookies()
+
+    driver.get(Config.contributionFrontend)
```

Comment:
Do we need to go to contributions, identity and support? Can't we just delete all *.thegulocal.com by getting one page?

Reply:
I feel it is safer to clean all the domains.